app/controllers/tmdb_controller.rb

```diff
@@ -134,53 +137,14 @@ def director_search
   end
 
   def discover_search
-    #format date/year hash passed in params. otherwise year is passed directly on paginated pages
-    params[:year] = params[:date][:year] if params[:date].present?
+    form_params = %i[sort_by date genre actor actor2 company mpaa_rating year_select page]
```

Comment:
I considered putting these params (and the ones on line 146) in a private method, but decided against it. This controller has a lot of methods, so I wanted to encapsulate everything related to this method in this single space.